### ISSUE TITLE
NVDA does not announce the Expand/Collapse action on the grouped DetailedList

### ISSUE THREAD

**Issue by mosesyap** [microsoft/fluentui#17203]
### Environment Information

- **Package version(s)**: 7.155.3
- **Browser and OS versions**: chrome/Edge Chromium/Win 10

### Describe the issue:

This repros on the samples in https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped.
NVDA does not announce that you can expand and collapse the children when the context is on the header row.  It simple says the state of the row.

### Please provide a reproduction of the issue in a codepen:
Can reproduce on the samples itself on the fluentui website

#### Actual behavior:

NVDA is not narrating the role and descriptive  name i.e. 'Collapse all/Expand all' for the button present in the table. 

Ex: when navigated to 'Expand all rows button', NVDA is narrating as 'Row 1 collapsed'. 

  
#### Expected behavior:

NVDA should narrate the role and descriptive  name i.e. 'Collapse all/Expand all' for the button present in the table. 

Ex: when navigated to 'Expand all rows button', NVDA should narrate as 'Row 1 expand all rows button collapsed'. 

### Documentation describing expected behavior

MAS Reference: 
MAS 4.1.2 - Name, Role, Value 
WCAG Reference: 
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by joschect** [comment]
@mosesyap Thanks for the submission! I'll need to investigate further but I suspect that this is an issue with NVDA as we have the appropriate aria values applied to the section toggle (see screenshot). 

![image](https://user-images.githubusercontent.com/20801176/109597479-e8a55b00-7acc-11eb-85fc-8bdad4e8da68.png)
